cmd/desync/cache.go

```diff
@@ -32,6 +34,8 @@ func cache(ctx context.Context, args []string) error {
 	flags.StringVar(&cacheLocation, "c", "", "use local store as cache")
 	flags.IntVar(&n, "n", 10, "number of goroutines")
 	flags.BoolVar(&desync.TrustInsecure, "i", false, "allow invalid certificates")
+	flags.StringVar(&clientCert, "clientCert", "", "Client Certificate for TLS authentication")
```

Comment:
Might be helpful to specify that this is expected to be a filename. ("Filename with client certificate [...]"?); please take this as given for the similar instances of this change elsewhere.